### ISSUE TITLE
GAMySQLDriver#execute: raises exception when int field is NULL

### ISSUE THREAD

**Issue by mbratch** [guillep/DBXTalk#8]
Using Pharo 5.0 and latest DBXTalk/Garage as of 6/1/2016.
Running on Windows 10 and connecting to a MySQL database on a Linux server.

Setup the following table:

```
mysql> select * from test_table;
+----+------+
| id | foo  |
+----+------+
|  1 |    1 |
|  2 | NULL |
+----+------+
mysql> describe test_table;
+-------+---------+------+-----+---------+----------------+
| Field | Type    | Null | Key | Default | Extra          |
+-------+---------+------+-----+---------+----------------+
| id    | int(11) | NO   | PRI | NULL    | auto_increment |
| foo   | int(11) | YES  |     | NULL    |                |
+-------+---------+------+-----+---------+----------------+
2 rows in set (0.01 sec)
```

Execute the following:

```
c := GADriver fromConnectionString: 'MySQL://192.168.1.100:3306/mydb?user=myname&password=mypassword'.
c connect.
q := c execute: 'select * from test_table'.
```

This results in the following error from GAMysqlDriver down in the NumberParser:

`Reading a number failed: a digit between 0 and 9 expected`

If the query is modified to avoid the NULL entries, then it works with no error:

```
q := c execute: 'select * from test_table where not (foo is null)'.
```

So it appears that the driver does not expect a NULL value for an integer.

**Comment by eMaringolo** [comment]
Couldn't reproduce it in Ubuntu server/image:

```
Gofer it
    smalltalkhubUser: 'DBXTalk' project: 'Garage';
    configurationOf: 'Garage';
    load.

#ConfigurationOfGarage asClass project stableVersion load: 'mysql'.

c := GADriver fromConnectionString: 'MySQL://localhost:3306/sodbxtest?user=sodbxtest&password=sodbxtest'.
c connect.

c execute: 'CREATE TABLE `test_table` (
    `id` INT NULL DEFAULT NULL,
    `foo` INT NULL DEFAULT NULL
)'.
c execute: 'insert into test_table values (1, 1)'.
c execute: 'insert into test_table values (2, NULL)'.

(c execute: 'select * from test_table') rows.

"c execute: 'drop table test_table'".
```

**Comment by mbratch** [comment]
I'll try your exact test tonight. I am running in Windows 10.

**Comment by mbratch** [comment]
Well, I'm a little embarrassed. This was my fault.

I had just started using DBXTalk/Garage and followed the instructions 
for installation on the website 
(https://guillep.github.io/DBXTalk/garage/installation.html). The 
installation shows version 0.1. I could not find information anywhere on 
what the latest version is, so I blindly followed the instructions.

I tried a couple of other versions and ended up installing 0.3 (I don't 
know if there's a later version, but 0.4 failed). This new version does 
not have the issue I cited.

I apologize for the error.

Mark
